debian/control

```diff
@@ -16,15 +16,15 @@ Build-Depends: debhelper-compat (= 13),
                gtk-doc-tools,
                libadwaita-1-dev (>= 1.0.1),
                libappstream-dev (>= 0.14.0),
-               libflatpak-dev (>= 1.14.0) [amd64 arm64 armel armhf i386 mipsel mips64el ppc64el s390x hppa powerpc ppc64 riscv64 x32],
+               libflatpak-dev [amd64 arm64 armel armhf i386 mipsel mips64el ppc64el s390x hppa powerpc ppc64 riscv64 x32],
```

Comment:
I was a little sceptical of this patch so I went looking for why Debian raised the dependency.

Here's the commit: https://salsa.debian.org/gnome-team/gnome-software/-/commit/562eb7ea73e362e561e154deefe5d5bf954519df

> ### Update dependency versions
> in preparation for the libsoup3 transition

Flatpak 1.14 introduced a curl backend…

Reply:
Correct. I've added this to the commit message. We will be able to drop this patch later.